src/app/public/modules/filter/filter-button.component.scss

```diff
@@ -6,6 +6,16 @@
    padding: 5px 11px; /* offset the thicker border to avoid button size change */
 }
 
+@include sky-theme-modern {
+  .sky-filter-btn-active, .sky-filter-btn-active:hover, .sky-filter-btn-active:focus {
```

Comment:
active state should use 1px action blue color border like checkbox does. hover and focus states should use regular button hover/focus styling.